CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changes to Calva.
 
 ## [Unreleased]
 
+- [Fix breakage with notebooks containing CRLF newlines on MS-Windows](https://github.com/BetterThanTomorrow/calva/issues/1994)
```

Comment:
```suggestion
- Fix: [Breakage with notebooks containing CRLF newlines on MS-Windows](https://github.com/BetterThanTomorrow/calva/issues/1994)
```

This is the format we usually use for these.

Reply:
Updated as suggested, but now using the title of the issue as link desc.